src/scripts/config.js

```diff
@@ -1,6 +1,9 @@
 (function () {
   'use strict';
 
+  var coachBasePath = '//openstax.github.io/concept-coach/';
```

Comment:
* yes
* i will need to check, and we can look into doing that, but i think that's secondary to getting coach to not need a hash update in cnx right?